UnitTests/PaymentsCoreTests/APIClient_Tests.swift

```diff
@@ -48,3 +48,4 @@ class APIClient_Tests: XCTestCase {
         XCTAssertEqual(response, "sample_id")
     }
 }
+// swiftlint:enable force_unwrapping implicitly_unwrapped_optional
```

Comment:
Do we need these at the end of every file?

Reply:
Yes! The swiflint github actions task was failing on every PR because the new swiftlint rule says that you for every `disable` rule at the top of a class, there has to be an `enable` at the end. Unless, you do a `disable:next` which disables a rule for the next line. But in the case of this tests, we `force_unwrap` many times, so its better to `disable` it at the begining an `enable` it at the end. 
This is code merged from another branch, so that swift lint passes. Please check #138 that addresses all of this `swiftlint` issues.

Reply:
We could potentially make unit test lint rules less strict to prevent having to do this a lot.

Reply:
Yeh, we have a ticket to relax linting rules in the tests - [DTNOR-628.](https://paypal.atlassian.net/browse/DTNOR-628) Just a matter of making time for it

Reply:
> new swiftlint rule says that you for every disable rule at the top of a class, there has to be an enable at the end

Could we just disable this new rule entirely in our [swiftlint.yml](https://github.com/paypal/iOS-SDK/blob/main/.swiftlint.yml)?

Reply:
@scannillo yes we can, do you want me to disable it across the project? or just for unitests? in order to disable it just for tests, i have to create a .yaml inside that specific folder

Reply:
@scannillo in order to not pollute this PR, disabled some lint rules for unitests in #141 addressing [DTNOR-628](https://paypal.atlassian.net/browse/DTNOR-628)